docs/walkthrough.md

```diff
@@ -193,6 +193,15 @@ kubectl config set-cluster service-catalog --server=http://$SVC_CAT_API_SERVER_I
 kubectl config set-context service-catalog --cluster=service-catalog
 ```
 
+Note: if you are using Google cloud or other cloud providers, you may need to
+setup proper firewall rules to allow your traffic get in.
+On Google cloud, you need to run the following commands:
```

Comment:
Move this and the following to a new toplevel `#troubleshooting` section at the end.